torch/_export/passes/lift_constants_pass.py

```diff
@@ -149,10 +149,18 @@ def lift_constants_pass(
 
     first_user_input_loc, first_user_input = 0, next(iter(gm.graph.nodes))
```

Comment:
Wouldn't this `next(iter(gm.graph.nodes))` just give you the first non-placeholder node?

Reply:
Hmm this won't work if there are param buffers i think. Because you do need to skip some placeholders.

Reply:
how come we don't just skip all the placeholders